pyoptsparse/pyOpt_variable.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         self.name = name
         self.type = type
         self.scalar = scalar
-        self.choices = None
+        self.choices = choices
```

Comment:
This line is repeated on line 53, I guess we can just keep one?

Reply:
Oh yeah L53 should be removed.